.github/workflows/action-lint.yml

```diff
@@ -0,0 +1,13 @@
+name: Lint GitHub Action
+on: [push]
+jobs:
+  action-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download actionlint
+        run: |
+          curl -fsSL https://github.com/rhysd/actionlint/releases/download/v1.5.3/actionlint_1.5.3_linux_386.tar.gz -o actionlint.tgz
```

Comment:
```suggestion
          curl -fsSL https://github.com/rhysd/actionlint/releases/download/v1.5.3/actionlint_1.5.3_linux_mad64.tar.gz -o actionlint.tgz
```

Reply:
Would also be nice to move the version number out to an env for easier updating

Reply:
i saw the build script but figured it would be nice. env var would be nice

Reply:
til 386 refers to the amd 32 bit version vs amd64 which is obviously for amd 64 bit systems